src/Models/StageModel.php

```diff
@@ -133,6 +133,30 @@ public function updateAllStages(array $stages) : bool
         }
     }
 
+    public function createOption(string $option, int $stageId) : bool
+    {
+        $query = $this->db->prepare("INSERT INTO `options` (`option`, `stageId`) VALUES (:optionText, :stageId);");
+        $query->bindParam(':optionText', $option);
+        $query->bindParam(':stageId', $stageId);
+        return $query->execute();
+    }
+
+    public function updateOption(string $option, int $optionId) : bool
+    {
+        $query = $this->db->prepare("UPDATE `options` SET `option` = :optionText WHERE `id` = :id");
+        $query->bindParam(':id', $optionId);
+        $query->bindParam(':optionText', $option);
+        return $query->execute();
+    }
+
+    public function deleteOption(int $optionId) : bool
+    {
+        $query = $this->db->prepare("UPDATE `options` SET `deleted` = '1' WHERE `id` = :optionId");
+        $query->bindParam(':optionId', $optionId);
+        return $query->execute();
+    }
+
+
```

Comment:
Can't put this anywhere, but just for your information: this commit was failing the CircleCI tests for a bit of a silly reason, which is that CircleCI expects every file to end with a "new line" character. Keep that in mind going forward.

Reply:
ok fixed thanks